index.html

```diff
@@ -6,101 +6,121 @@
     <link href="//fonts.googleapis.com/css?family=Roboto:400,500,700,400italic" rel="stylesheet" type="text/css">
     <link href="//fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" type="text/css">
     <link rel="stylesheet" type="text/css" href="www/css/material.css">
-    <title>Material icons guide - Google design</title>
+    
```

Comment:
This file is generated from Markdown on our end. I've captured your changes, and I'll port them back. Mind reverting this file?